MicroPython/src/common/utils.py

```diff
@@ -171,26 +171,28 @@ def get_wifi_and_cloud_handlers(sync_time: bool = False) -> (WirelessConnectionC
         except Exception:
             logging.error("Error in disconnecting WiFi controller")
 
-        logging.debug("RESETTING BOARD")
-        machine.reset()
+        logging.debug("sleep({})".format(config.cfg.data_publishing_period_in_ms))
+        machine.deepsleep(config.cfg.data_publishing_period_in_ms)
```

Comment:
Is it the same code snippet as the one called after successfully sent data to a cloud (log that it's going to sleep and sleep for publishing period in ms)? If so - add some shorty comment on what is going on here.